execution_engine/src/runtime/mod.rs

```diff
@@ -994,6 +994,21 @@ where
                 CLValue::from_t(()).map_err(Self::reverter)
             })(),
 
+            auction::METHOD_TRANSFER_VALIDATOR => (|| {
```

Comment:
The naming needs to change for the following reasons:
1. Transfer has an established different meaning in this system, and we should not introduced semantic confusion.
2. This feature updates a bid record, regardless of whether that bid record correlates to a current validator, a historical validator, a want-to-be-a validator, or a never-has-been-a validator.

I would recommend calling it what it is:  change_bid_public_key

Reply:
Good point. Change the naming and removed all references to "current validator" etc.